tutorials/tmva/TMVA_SOFIE_RDataFrame.C

```diff
@@ -0,0 +1,57 @@
+/// \file
+/// \ingroup tutorial_tmva
+/// \notebook -nodraw
+/// This macro provides an example of using a trained model with Keras
+/// and make inference using SOFIE and RDataFrame
+/// This macro uses as input a Keras model generated with the
+/// TMVA_Higgs_Classification.C tutorial
+/// You need to run that macro before to generate the trained Keras model
+/// Then you need to run the macro TMVA_SOFIE_Keras_HiggsModel.C to generate the  corresponding
+/// header file using SOFIE.
+///
+/// Execute in this order:
+/// ```
+/// root TMVA_Higgs_Classification.C
+/// root TMVA_SOFIE_Keras_HiggsModel.C
+/// root TMVA_SOFIE_RDataFrame.C
+/// ```
+///
+/// \macro_code
+/// \macro_output
+/// \author Lorenzo Moneta
+
+using namespace TMVA::Experimental;
+
+// need to add the current directory (from where we are running this macro)
+// to the include path for Cling
+R__ADD_INCLUDE_PATH($PWD)
+#include "Higgs_trained_model.hxx"
+#include "TMVA/SOFIEHelpers.hxx"
+
+using namespace TMVA::Experimental;
+
+void TMVA_SOFIE_RDataFrame(){
+
+   std::string inputFileName = "Higgs_data.root";
+   std::string inputFile = "http://root.cern.ch/files/" + inputFileName;
+
+    int nthreads = ROOT::GetThreadPoolSize();
```

Comment:
although at the moment the numbers should always correspond, it would be safer to use `df1.GetNSlots()` as the number of slots (in the future number of RDF slots might be decoupled from the total size of the ROOT thread pool)